multimodal/src/autogluon/multimodal/models/huggingface_text.py

```diff
@@ -254,3 +254,10 @@ def get_layer_ids(self):
             name_to_id[n] = 0
 
         return name_to_id
+
+    def dump_weights_and_config(self, save_path: str = "./", tokenizers: Optional[dict] = None):
```

Comment:
Not sure whether `dump_weights_and_config ` is a good name. Using `dump` in predictor is fine since predictor contains many stuff. Here in the context of one model, maybe `save` is a better keyword.

Reply:
Shall we name it as `save` or `save_weights_and_config`

Reply:
`save` sounds good. We can add arguments for this function if we customize what we want to save.